src/sass/components/_network.scss

```diff
@@ -51,6 +60,49 @@ $mobile-btn-padding: 52px;
   }
 }
 
+.endorsement {
```

Comment:
When naming styles, think in terms of "searchability". With a system as big as We Vote, I try to use names that are unique and can be found easily. So instead of using a style named "endorsement", please use something like "endorsement-card".